site/components/src/main/java/uk/nhs/hee/web/beans/News.java

```diff
@@ -74,4 +73,14 @@ public List<HippoBean> getAuthors() {
     public Boolean getHideAuthorContactDetails() {
         return getSingleProperty("hee:hideAuthorContactDetails");
     }
+
+    @HippoEssentialsGenerated(internalName = "hee:globalTaxonomyProfessions")
```

Comment:
Could we replace these taxonomy-based field methods with the following to return `TaxonomyClassification` like how we did on `uk.nhs.hee.web.beans.PublicationLandingPage` for ease of rendering in the freemarker template in future ?
```
    @HippoEssentialsGenerated(internalName = "hee:globalTaxonomyProfessions", allowModifications = false)
    public TaxonomyClassification getGlobalTaxonomyProfessions() throws RepositoryException {
        return TaxonomyTemplateUtils.getTaxonomyClassification(
                this.getNode(),
                "hee:globalTaxonomyProfessions",
                HEETaxonomy.HEE_GLOBAL_PROFESSIONS);
    }

    @HippoEssentialsGenerated(internalName = "hee:globalTaxonomyHealthcareTopics", allowModifications = false)
    public TaxonomyClassification getGlobalTaxonomyHealthcareTopics() throws RepositoryException {
        return TaxonomyTemplateUtils.getTaxonomyClassification(
                this.getNode(),
                "hee:globalTaxonomyHealthcareTopics",
                HEETaxonomy.HEE_GLOBAL_HEALTHCARE_TOPICS);
    }
```